borg/testsuite/archiver.py

```diff
@@ -71,6 +71,29 @@ def __exit__(self, *args, **kw):
                 os.environ[k] = v
 
 
+def exec_cmd(*args, archiver=None, fork=False, **kw):
```

Comment:
i don't understand why this extra wrapper is created here. why not cleanup the signature of the `cmd` function and all callers?